api/projects.ts

```diff
@@ -347,31 +348,36 @@ export async function listAppsForMigration(
   });
 }
 
-export async function beginMigration(
+function mapPlatformVersionToEnum(platformVersion: string): string {
```

Comment:
Would we be future proofing ourselves by putting this logic somewhere else? If we limit the amount of logic that happens in these api utils, we won't have to worry about making (potentially breaking) logic changes down the road.

Or are these enums very specific to the migrations api?

Reply:
> Or are these enums very specific to the migrations api?

Yeah, they seem to be specific to migrations ATM.  This is the only place we are using it ATM.  Since this particular method isn't exposed, we could move it in the future if we find a reason to reuse it.